packages/augur-core/source/contracts/Cash.sol

```diff
@@ -54,11 +54,11 @@ contract Cash is ITyped, ICash, ICashFaucet {
     }
 
     function transferFrom(address _from, address _to, uint256 _amount) public returns (bool) {
-        uint256 _allowance = allowed[_from][msg.sender];
-        require(_amount <= _allowance, "Not enough funds allowed");
-        if (_allowance != ETERNAL_APPROVAL_VALUE) {
-            allowed[_from][msg.sender] = _allowance.sub(_amount);
-        }
+        // uint256 _allowance = allowed[_from][msg.sender];
```

Comment:
Won't the predicate still be dealing with real DAI on the main chain? Thats what this contract is meant to stand in for.